mods/cnc/maps/nod10a/rules.yaml

```diff
@@ -2,7 +2,7 @@ World:
 	LuaScript:
 		Scripts: campaign.lua, utils.lua, nod10a.lua
 	MissionData:
-		Briefing: GDI is developing an orbital weapon. Our spies have told us of a large lake near the location of the R&D center.\n\nFind the base, and use the sniper to eliminate their scientist.
+		Briefing: GDI is developing an orbital weapon. Our spies have identified a large lake near the R&D center.\n\nFind the base and use the sniper to take out their lead scientist.
```

Comment:
This doesn't really make sense (why is the lake relevant?). This could be "Our spies have located the R&D center near a large lake" instead.